File: phase5/cross_language/corpus/python/fibonacci.py
def fibonacci(n):
    """Return nth Fibonacci number (0-indexed). fib(0)=0, fib(1)=1."""
    if n < 0:
        raise ValueError("n must be non-negative")
    if n == 0:
        return 0
    a, b = 0, 1
    for _ in range(1, n):
        a, b = b, a + b
    return b

File: phase5/cross_language/corpus/java/fibonacci.java
public class Fibonacci {
    public static long fibonacci(int n) {
        if (n < 0) throw new IllegalArgumentException("n must be non-negative");
        if (n == 0) return 0;
        long a = 0, b = 1;
        for (int i = 1; i < n; i++) {
            long temp = b;
            b = a + b;
            a = temp;
        }
        return b;
    }
}
